### PR TITLE
Added cli commands to get function cluster related information

### PR DESCRIPTION
**Pull request by srkukarni**
### Motivation

Info like all workers in the cluster, cluster leader and function-assigments already had a endpoint, but there was no cli support. This pr adds them. We also refactor some of such 'cluster' type of commands into their own endpoint

### Modifications

Describe the modifications you've done.

### Result

After your change, what will change.

### CODE REVIEW

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Worker.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 403, message = "The requester doesn't have admin permissions")
+
+    })
+    @Path("/cluster")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getCluster() {
+        try {
+            // log.info("SANJEEV CAME1");
```
Comment (rdhabalia) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Worker.java]: need to remove

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/WorkerStats.java [hunk 2]
```diff
@@ -1,56 +0,0 @@
...
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
-import lombok.extern.slf4j.Slf4j;
-
-@Slf4j
-@Path("/worker-stats")
-public class WorkerStats extends FunctionApiResource {
-
-    @GET
-    @Path("/functions")
```
Comment (rdhabalia) [inline pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/WorkerStats.java]: I don't see this API added anywhere?

File: pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/Resources.java [hunk 3]
```diff
@@ -37,7 +36,7 @@ private Resources() {
         return new HashSet<>(
                 Arrays.asList(
                         FunctionApiV2Resource.class,
-                        WorkerStats.class,
+                        WorkerApiV2Resource.class,
```
Comment (rdhabalia) [inline pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/Resources.java]: why do we name it v2? we don't have v1 for functions ?

File: pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/WorkerStats.java (hunk 2 above)
Comment (srkukarni) [reply pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/WorkerStats.java]: An oversight on my part while moving. I've added it but moved to /worker/functionsmetrics to better reflect what the call does

File: pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/Resources.java (hunk 3 above)
Comment (srkukarni) [reply pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/rest/Resources.java]: /worker is strictly v2. I see no point in adding a v1.

**Comment by srkukarni** [comment]
@rdhabalia I've corrected the mistake of taking out functionsmetrics.
wrt having a seperate endpoint, I'm not sure if its really needs to go in /worker-stats? /worker/metrics and /worker/functionsmetrics should do the same job?

**Comment by srkukarni** [comment]
run integration tests

**Comment by srkukarni** [comment]
run integration tests

**Comment by rdhabalia** [comment]
> wrt having a seperate endpoint, I'm not sure if its really needs to go in /worker-stats?

actually, motivation behind having separate endpoint `workers` and `worker-stats` is `worker-stats` will always serve localhost stats and this resource can't redirect or serve for other workers. and that was the reason we have broker-stats to serve broker's local resources. so, I thought we can keep that consistency. @merlimat any thoughts?

**Comment by srkukarni** [comment]
@rdhabalia Currently /workers doesn't redirect either. /functions might.

**Comment by srkukarni** [comment]
run cpp tests

**Comment by srkukarni** [comment]
@rdhabalia I have seperated out the stats calls into their own endpoint very similar to broker. Please take a look again.

**Pull request merged**